meta-ov/recipes-kernel/linux/files/0004-1.1V-fix-for-sun7i-a20.dtsi.patch

```diff
@@ -0,0 +1,39 @@
+From 7e668bb8ed7ca437c2758de490162bfc33c67ab5 Mon Sep 17 00:00:00 2001
+From: Torsten Beyer <tb@pobox.com>
+Date: Tue, 19 Jul 2022 10:18:57 +0200
+Subject: [PATCH 4/4] 1.1V fix for sun7i-a20.dtsi
+
+Signed-off-by: Torsten Beyer <tb@pobox.com>
+---
+ arch/arm/boot/dts/sun7i-a20.dtsi | 8 ++++----
+ 1 file changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/arch/arm/boot/dts/sun7i-a20.dtsi b/arch/arm/boot/dts/sun7i-a20.dtsi
+index 5574299685ab..7cba57b5eab0 100644
+--- a/arch/arm/boot/dts/sun7i-a20.dtsi
++++ b/arch/arm/boot/dts/sun7i-a20.dtsi
+@@ -113,8 +113,8 @@
+ 				<864000	1300000>,
+ 				<720000	1200000>,
+ 				<528000	1100000>,
+-				<312000	1000000>,
+-				<144000	1000000>;
++				<312000	1100000>,
++				<144000	1100000>;
```

Comment:
I wonder if you can just override those values in one of dts-override files that OV uses. Would it make more sense than patching linux kernel?

Reply:
Upon further thought: the original dtsi file of the linux kernel is _wrong_, really. At least with respect to what the A20 SDK says in relation to OPP voltages. Hence it is probably even more logical to patch the dtsi file, too.

Reply:
Oh, you indeed addressed this in the description (I should've read it first). It makes sense!

Reply:
I wonder if it would also make sense to contribute this change straight to linux upstream?

Reply:
It would and I have posted it to the maintainers and the kernel mailinglist bots  (which rejected the mail, though)...let's see how/if they want to consider input from gliding underlings :-)